DataFormats/HGCalReco/interface/Trackster.h

```diff
@@ -20,7 +20,7 @@ namespace ticl {
   public:
     typedef math::XYZVector Vector;
 
-    enum IterationIndex { TRKEM = 0, EM, TRKHAD, HAD, MIP };
+    enum IterationIndex { TRKEM = 0, EM, TRKHAD, HAD, MIP, SIM, SIM_CP };
```

Comment:
can you comment if and how this change to the dataformat affects backwards compatibility?